services/ui-src/src/components/fields/ChoiceListField.tsx

```diff
@@ -171,18 +144,52 @@ export const ChoiceListField = ({
     }
   };
 
-  // update field values
+  const getNestedChildFieldsOfUncheckedParent = (choices: FieldChoice[]) => {
+    // set up nested field compilation
+    const nestedFields: any = [];
+    const compileNestedFields = (fields: FormField[]) => {
+      fields.forEach((field: FormField) => {
+        // for each child field, get field info
+        const fieldDefaultValue = ["radio", "checkbox"].includes(field.type)
+          ? null
+          : "";
+        const fieldInfo = {
+          name: field.id,
+          shouldClear: true,
```

Comment:
Bug: Is saving regardless of field value. Might already be cleared and therefore not need to save.
```suggestion
          shouldClear: form.getValues(field.id) !== fieldDefaultValue,
```